orchestrator/modules/tools/discovery/handlers_watches.py

```
@@ -0,0 +1,426 @@
+            created_by=created_by,
+            owner_agent_id=owner_agent_id,
+            success_criteria=success_criteria,
+        )
+        logger.info(
+            "[Watches] auto-created watch %s on %s:%s",
+            watch.id,
+            target_type,
+            target_id,
+        )
+        return watch
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)